### PR TITLE
Medical HUD Upgrade

### PR DESCRIPTION
**Pull request by Puckaboo2**
The medical HUD now indicates health in increments of ten. I overhauled the sprite of the HUD's health indicator to be 20 pixels, in which two pixels deducted count in a decimation of health.

http://i67.tinypic.com/k2l8jl.png

The HUD can also display negative values in increments of ten, in which deducted health appears black and current health is a flashing red. It even flashes faster and faster beyond -70% health and even has a flashing death indicator at -90%! I updated the code to match the values of the icon states, accordingly.

tl;dr All I did was add and change values based on the icon states I created. It works.

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
Sprite PR without sprites.

**Comment by Puckaboo2** [comment]
Huh? The medical HUD file is in the pull request.

**Comment by Spacemanspark** [comment]
Yes, but generally you'd put pictures of the new sprites in the OP, for convenience sake.

**Comment by Puckaboo2** [comment]
Apologies! A picture is available, now.

**Comment by FlattestGuitar** [comment]
Looks dope to me! 👌

**Comment by Fox-McCloud** [comment]
Shouldn't it be the `dead` bar that has the skull and not the "just before death". That's going to cause some confusion.

File: code/game/data_huds.dm [hunk 1]
```diff
@@ -68,45 +68,53 @@
...
+			return "health30"
+		if(20 to 30)
+			return "health20"
+		if(10 to 20)
+			return "health10"
+		if(0 to 10)
 			return "health0"
-		if(-85 to -50)
+		if(-10 to 0)
+			return "health-0" //Hard crit begins here. The health bar will turn a brilliant red as usual, but deducted health will be black.
```
Comment (Fox-McCloud) [inline code/game/data_huds.dm]: Incorrect. Soft-crit begins here. Hard crit begins at -50.

**Comment by Puckaboo2** [comment]
If I were to remove that comment and remove the flashy skull, will you merge the changes?

**Comment by Crazylemon64** [comment]
Ah, why did you close this one? It seemed fine to me

**Comment by Puckaboo2** [comment]
I forgot git amend existed and I ended up messing up and removing local commits that weren't even mine, so I reset. 

Dw, I've got my shiz backed up.
